### PR TITLE
Nb/30.04.2019 Changes deployed 29.04.2019

### PR DESCRIPTION
**Pull request by bogyo210**

### CODE REVIEW

File: changelog_5c1ce29804286304a71d9ba4.md [hunk 1]
```diff
@@ -2,6 +2,17 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of Apr 29, 2019
```
Comment (markoa) [inline changelog_5c1ce29804286304a71d9ba4.md]: Should be Apr 22

File: changelog_5c1ce29804286304a71d9ba4.md [hunk 2]
```diff
@@ -2,6 +2,17 @@ Thank you for using Semaphore!
...
+
+- Platform updates:
+  - Chrome 74, ChromeDriver 74
+  - Heroku 7.24.1
+  - Git-lfs 2.7.2
+  - Pip 19.1
+  - Phpunit 7.5.9
+  - Removed Oracle Java 7,9,10
+  - Java 8,11 changed from Oracle Java to openjdk
```
Comment (markoa) [inline changelog_5c1ce29804286304a71d9ba4.md]: Please unify Java changes in a single bullet point, adding a point that there's nothing users need to do.

File: ubuntu-1804-image_5b461f730428630abc0bf442.md [hunk 3]
```diff
@@ -213,14 +212,7 @@ Available versions:
...
 
-Rust 1.33 is available via rustup, with appropriate language tools including:
-
-- cargo
-- rustc
-- rustdoc
-- rust-gdb
-- rust-lldb
-- rustup
+Available via docker image
```
Comment (markoa) [inline ubuntu-1804-image_5b461f730428630abc0bf442.md]: This is not helpful to end users. Let's just remove Haskell and Rust sections while we don't have any examples.

File: changelog_5c1ce29804286304a71d9ba4.md (hunk 1 above)
Comment (bogyo210) [reply changelog_5c1ce29804286304a71d9ba4.md]: Changed

File: changelog_5c1ce29804286304a71d9ba4.md (hunk 2 above)
Comment (bogyo210) [reply changelog_5c1ce29804286304a71d9ba4.md]: Unified

File: ubuntu-1804-image_5b461f730428630abc0bf442.md (hunk 3 above)
Comment (bogyo210) [reply ubuntu-1804-image_5b461f730428630abc0bf442.md]: Rust and haskell removed from file

**Pull request merged**